### PR TITLE
AU eRequesting ServiceRequest - add Service.code implementation guidance

### PR DESCRIPTION
**Pull request by dbojicic**
Add profile specific implementation guidance re ServiceRequest.code bindings agreed by the TDG (https://confluence.hl7.org/pages/viewpage.action?pageId=256181856&preview=/256181856/256183159/AUeRequesting%20TDG%2004%20JUL%2024.pdf). 

_Note: Changes related to adding the General Guidance page have been excluded from this PR and will be provided in a separate PR._

### CODE REVIEW

**Comment by angusmillar** [comment]
Fumbling my way through the process here. I have reviewed the PR and I'm generally happy with its contents.

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    - if a suitable code from the preferred value set is not available, **SHOULD** use a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set 
+    - if a suitable code from SNOMED CT is not available, **SHOULD** use a local code 
+    - **SHOULD** use a code over sending free text in `Condition.code.text` when appropriate codes are available
+  - When representing an imaging order (`ServiceRequest.category` is SNOMED CT 363679005 |Imaging), `ServiceRequest.code`:
+    - **SHOULD** use a code from the preferred RANZCR Radiology Referral value set
+    - if a suitable code from the preferred value set is not available, **SHOULD** use a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set
+    - if a suitable code from SNOMED CT is not available, **SHOULD** use a local code
+    - **SHOULD** use a code over sending free text in `Condition.code.text` when appropriate codes are available
```
Comment (dt-r) [inline input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:15]: Add in a line about use of text as a 1st level point. Suggest:
- The use of coding can vary significantly, implementers across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+### Usage scenarios
+TBD
+
+### Profile specific implementation guidance
+- `ServiceRequest.code` represents the type of service being requested or ordered, such as a diagnostic test or a procedure. Depending on the context provided by `ServiceRequest.category`, `ServiceRequest.code` **SHOULD** be selected from a corresponding domain-specific value set:
```
Comment (dt-r) [inline input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:5]: The first point can be removed, and allow for content on pathology and imaging to be 1st level points.

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - When representing a pathology order (`ServiceRequest.category` is SNOMED CT 108252007 |Laboratory procedure), `ServiceRequest.code`:
+    - **SHOULD** use a code from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set
```
Comment (dt-r) [inline input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:7]: Suggest we could rephrase this to be simpler and more active language:
- Pathology orders are represented as:
   - `ServiceRequest.category` = "108252007" (Laboratory procedure)
   - `ServiceRequest.code`:
       - **SHOULD** be selected from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set
       - if a system is unable to provide a code from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set it is recommended to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set.
       - if a system is unable to provide a code from either the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set or the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set then it is preferred that a local code is provided over only free text

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    - if a suitable code from the preferred value set is not available, **SHOULD** use a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set 
+    - if a suitable code from SNOMED CT is not available, **SHOULD** use a local code 
+    - **SHOULD** use a code over sending free text in `Condition.code.text` when appropriate codes are available
+  - When representing an imaging order (`ServiceRequest.category` is SNOMED CT 363679005 |Imaging), `ServiceRequest.code`:
```
Comment (dt-r) [inline input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:11]: Additionally suggest that the guidance on coding become Comments on the element in the profile with some slight rewording.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:15]: Done

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:5]: Done

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:7]: Updated

File: input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md (hunk 4 above)
Comment (dbojicic) [reply input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md:11]: Done

**Comment by dbojicic** [comment]
implementation guidance has been added via #44.
